RELEASE/scripts/autoscend/iotms/mr2018.ash

```diff
@@ -1270,7 +1271,7 @@ boolean auto_voteMonster(boolean freeMon, location loc, string option)
 	{
 		return false;
 	}
-	if(!possessEquipment($item[&quot;I voted!&quot; sticker]))
+	if(!possessEquipment($item[&quot;I voted!&quot; sticker]) || !auto_is_valid($item[&quot;I voted!&quot; sticker]))
```

Comment:
Should this also be `is_unrestricted`? I think some things with weird abilities like the voted sticker work in g-lover regardless.